ecommerce/api.py

```diff
@@ -110,14 +106,9 @@ def determine_visitor_country(request: HttpRequest or None) -> str or None:
     try:
         client_ip, _ = get_client_ip(request)
     except TypeError:
-        return None
-
-    ip_country_code = ip_to_country_code(client_ip)
-
-    if ip_country_code == profile_country_code:
-        return ip_country_code
+        return profile_country_code
 
-    return None
+    return ip_to_country_code(client_ip) or profile_country_code
```

Comment:
Let's assume that the profile country is India and the user is using VPN. We won't be charging the tax in this case. Do we need to handle this case?

CC: @pdpinch

Reply:
This is actually the scenario that triggered these changes - customers in India connecting via a US-based VPN. The code will prefer to charge tax based on the IP location but will fall back to the profile location if there's no tax assessed.

In practice: we don't assess tax in the US (really, any country but India). So, the call to determine the country tax by IP returns None if they're on the VPN and so they are charged based on their profile country. We do want to make sure that if they're using an India-based IP that they are charged tax, though, hence the preference for IP location.